common/src/main/java/eu/europeana/fulltext/entity/AnnoPage.java

```diff
@@ -83,19 +99,27 @@ public void setTgtId(String tgtId) {
     }
 
     public List<Annotation> getAns() {
-        return ans;
+        return new ArrayList<>(ans);
     }
 
     public void setAns(List<Annotation> ans) {
-        this.ans = ans;
+        this.ans = new ArrayList<>(ans);
     }
 
     public Date getModified() {
-        return modified;
+        return (Date) modified.clone();
     }
 
     public void setModified(Date modified) {
-        this.modified = modified;
+        this.modified = (Date) modified.clone();
+    }
+
+    public String getLang() {
+        return Objects.requireNonNullElse(lang, "x");
+    }
```

Comment:
Wouldn't it be better to simply return the lang value? Suppose there's a record without a lang field, then we have a null value and the field consequently won't be displayed in the output. Personally I find that more elegant than displaying an 'x'?